packages/cloud-cognitive/src/components/SidePanel/SidePanel.js

```diff
@@ -583,11 +585,11 @@ export let SidePanel = React.forwardRef(
               animation: `${
                 open
                   ? placement === 'right'
-                    ? 'sidePanelEntranceRight 250ms'
-                    : 'sidePanelEntranceLeft 250ms'
+                    ? `sidePanelEntranceRight ${moderate02}`
+                    : `sidePanelEntranceLeft ${moderate02}`
                   : placement === 'right'
-                  ? 'sidePanelExitRight 250ms'
-                  : 'sidePanelExitLeft 250ms'
+                  ? `sidePanelExitRight ${moderate02}`
+                  : `sidePanelExitLeft ${moderate02}`
```

Comment:
Should the animations 'sidePanelOverlayEntrance 250ms' and 'sidePanelOverlayExit 250ms' (forty lines further down) also be changed to use ${moderate02}?

Reply:
Yes, they should. Thanks for catching that.

Reply:
Updated